### DISCUSSION TITLE
How to build with develop branch of theme?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#558] [Q&A] [answered]
I tried to test current develop branch.

What I did:
```
cd themes/hugo-theme-bootstrap
<some commands I don't remember to pull develop branch>
npm install
```

When building, I get:

```
Error: Error building site: TOCSS: failed to transform "main/scss/main.css" (text/css): SCSS processing failed: file "/blog/themes/hugo-theme-bootstrap/assets/main/scss/_bootstrap.scss", line 7, col 1: File to import not found or unreadable: bootstrap/scss/maps.
```

Also did `npm install` in the blog main directory.

**Comment by razonyang** [comment, marked as answer]
You probably forgot the `hugo mod npm pack` step, that command merge the latest dependencies to your project. And then you'll need to execute `npm install` again.

I also added the **Testing** section in #547.
